### DISCUSSION TITLE
Documentation Team and propose new Label

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#1081] [Ideas]
We are using Oqtane for our enterprise app and I'm tasked with creating onboarding documentation for new developers.  I'd like to contribute public versions of this this back to the project as well as understand the gaps in the current documentation.  To facilitate this, I feel a "documentation" issue tag would be helpful.  That way we can add new documentation issues and allow the code contributors to filter them out.  We are using DocFX for the enterprise docs and we could use that here as well.  I'm just starting down this road, so any direction from the core contributors would be greatly appreciated.

**Comment by markdav-is** [comment]
I just found the docs repo!

**Comment by hishamco** [comment]
https://github.com/oqtane/oqtane.docs
